packages/fhir-cht/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@openfn/language-fhir-cht",
+  "version": "1.0.0",
+  "description": "OpenFn fhir-cht adaptor",
+  "scripts": {
+    "build": "pnpm clean && build-adaptor fhir-cht",
+    "build:src": "pnpm generate-fhir fhir-cht",
+    "build:spec": "pnpm generate-fhir fhir-cht --respec",
+    "test": "mocha --experimental-specifier-resolution=node --no-warnings",
+    "test:watch": "mocha -w --experimental-specifier-resolution=node --no-warnings",
+    "clean": "rimraf dist types docs",
+    "pack": "pnpm pack --pack-destination ../../dist",
+    "lint": "eslint src"
+  },
+  "author": "Open Function Group",
+  "license": "LGPLv3",
+  "type": "module",
+  "fhir": {
+    "spec": "https://hl7.fr/ig/fhir/core/2.0.0/definitions.json.zip",
+    "adaptorGeneratedDate": "2024-11-22T15:38:03.869Z",
+    "name": "hl7.fhir.fr.core",
+    "specVersion": "4.0.1",
```

Comment:
I'll look into this but this looks like a fhir version number, not the spec version number?